core/src/main/scala/io/finch/Error.scala.orig

```diff
@@ -0,0 +1,121 @@
+package io.finch
```

Comment:
Hi, that was not added by me.
And when I remove it I get a warning:
"Package names doesn't correspond to directories structure, this may cause problems with resolve to classes from this file.
Inspection for files with package statement which does not correspond to package structure"